### PR TITLE
merged conflict and added two more speakers

### PR DESCRIPTION
**Pull request by KarenZhou0**

### CODE REVIEW

File: components/Sponsors.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { H1, Section } from './style'
-import { Style as StyleLogoGrid } from './LogoGrid'
+import { Style as Style_LogoGrid } from './LogoGrid'
```
Comment (tianhuil) [inline components/Sponsors.tsx:3]: Please do not use `Style_LogoGrid`.  This is the wrong style of variable.

**Review by tianhuil — CHANGES_REQUESTED** [review]
It looks like you built on an old branch.  Can you build on the latest master?

File: components/Sponsors.tsx (hunk 1 above)
Comment (KarenZhou0) [reply components/Sponsors.tsx:3]: I used StyleLogoGrid first but I think you changed it to Style_LogoGrid, if it's the wrong style of variable, what should I use?

Comment (tianhuil) [reply components/Sponsors.tsx:3]: I didn't change the style to `Style_LogoGrid` -- you did in a previous PR.  Note that it is set to `Style_LogoGrid` in the current version and I don't know why you are changing it.

File: components/Sponsors.tsx [hunk 2]
```diff
@@ -13,17 +13,17 @@ export const SponsorComp = () => {
   return (
     <Section id="sponsors">
       <H1>Sponsors</H1>
-      <StyleLogoGrid>
+      <Style_LogoGrid>
         <Row>
-          {logos.map((logo, k) => (
-            <Col size={4} tablet={{ size: 2 }} key={k}>
+          {logos.map((logo) => (
```
Comment (tianhuil) [inline components/Sponsors.tsx:18]: The `key` argument is necessary per React: https://reactjs.org/docs/lists-and-keys.html

Please do not delete.

**Review by tianhuil — CHANGES_REQUESTED** [review]
I think you got confused in your merge.  Please just do this again from the latest master and don't make any changes that are you necessary.

**Pull request closed without merging**